lib/pages/chat/chat.dart

```diff
@@ -273,6 +257,21 @@ class ChatController extends State<Chat> {
     setState(() {});
   }
 
+  bool isContactsLastPage() {
+    final previousPath = VRouter.of(context).previousPath;
+    return previousPath?.endsWith('newprivatechat') == true 
+      || previousPath?.endsWith('contactsTab') == true
+      || previousPath?.endsWith('emptyChat') == true;
+  }
```

Comment:
could you check if previousPath is null please ? 

WDYT 
```suggestion
  bool isContactsLastPage() {
    final previousPath = VRouter.of(context).previousPath;

    if (previousPath == null) return false;
    return previousPath.endsWith('newprivatechat')
      || previousPath.endsWith('contactsTab')
      || previousPath.endsWith('emptyChat');
  }

```

Reply:
IMO, no need, because `previousPath` is nullable, with the null safety call `?`. https://dart.dev/codelabs/null-safety